### ISSUE TITLE
Operator indirectly uses forbidden and unlicensed code

### ISSUE THREAD

**Issue by kitd** [Apicurio/apicurio-registry-operator#101]
A dependency graph of the Apicurio operator shows it is using code from bou.ke/monkey which is unlicensed (see https://github.com/bouk/monkey/blob/master/LICENSE.md) and whose developer has forbidden its use.

Here's the tree:
```
github.com/Apicurio/apicurio-registry-operator@master pulls -> 
github.com/operator-framework/operator-sdk@v0.17.0 -> 
github.com/operator-framework/api@v0.1.1 pulls -> 
github.com/operator-framework/operator-registry@v1.5.3 pulls-> 
github.com/otiai10/copy@v1.0.1 pulls -> 
bou.ke/monkey@v1.0.1
```

I think the problem is really that the operator is using a fairly old version of the operator-sdk framework @ 0.17.0 . An upgrade to a later version would fix this.

**Comment by kitd** [comment]
After further investigation, it seems this is mitigated by upgrading the operator framework sdk version to 0.18.2. This would be a quick win IMO.

**Comment by jsenko** [comment]
@kitd hi, any updates on this issue? I've see that you closed the update PR.

**Comment by kitd** [comment]
Yes, apologies, my fix wasn't complete. 

I will prepare a better one and reopen the PR (or create a new one) when I get the chance.

Quick question though: I have had to make changes to introduce contexts into some of the client calls to allow for updated K8s APIs. I have used `context.TODO()` for now. Would you prefer `context.Background()` instead?

**Comment by jsenko** [comment]
This issue has been resolved, and will not affect the upcoming `1.0.0-v2.0.0.final` release. We have migrated to Operator SDK 1.4
